src/System.Private.Reflection.Metadata.Ecma335/src/Shims/Path.cs

```diff
@@ -0,0 +1,27 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+
+namespace System.IO
+{
+    internal static class Path
+    {
+        internal static char[] GetInvalidFileNameChars() => new char[]
+        {
+                    '\"', '<', '>', '|', '\0',
```

Comment:
Yeah I know. When we build this for the corert target which as far as I know hasn't been stood up yet, we won't include this shim, and it will work. Alternatively, when uapaot System.Private.CoreLib has a version of path it will trigger a build break here, and we'll remove this code.